src/main/java/com/jcabi/jdbc/JdbcSession.java

```diff
@@ -309,6 +310,25 @@ public void commit() throws SQLException {
     }
 
     /**
+     * Call an SQL stored procedure.
+     *
+     * <p>JDBC connection is opened and, optionally, closed by this method.
```

Comment:
@amihaiemil OK, let it be. But I still does not get `optionally` concept, can you explain?